CONTRIBUTING.md

```diff
@@ -45,3 +45,36 @@ To generate this package, run the following script:
 ```
 
 This will generate the source code for this repo.
+
+## Generating the Library (protobuf)
+
+```sh
+git clone https://github.com/googleapis/google-cloudevents-go
+cd google-cloudevents-go
+sh ./tools/setup-generator.sh
+export GENERATE_DATA_SOURCE=tmp/google-cloudevents
+export GENERATE_PROTOC_PATH=tmp/protobuf/bin/protoc
+sh ./generate-code.sh
+```
+
+### Manage the Data Source
```

Comment:
Thinking a sentence might be useful here to clarify that the below instructions are an alternative to the above instructions. I wasn't totally clear on this until I read through the code.